documentation/src/components/side-nav-search.ts

```diff
@@ -149,6 +149,15 @@ class SearchComponent extends LitElement {
         }
     }
 
+    preventDefault(event: Event): void {
```

Comment:
nit: this would be more descriptive as `handleSubmit` since it's now doing more than just preventing default actions.

Reply:
👍🏼